.bazelignore

```diff
@@ -1 +1,4 @@
+bazel-bin
+bazel-buildbuddy
+bazel-out
```

Comment:
FYI, this fixes an issue that running `bazel test //...` tries to pick up targets under these dirs. I don't think it's related to this PR, since I've had to make these changes in the past, but just never got around to checking them in.